### ISSUE TITLE
Adjusting paths

### ISSUE THREAD

**Issue by arnaldop** [karatelabs/karate-intellij-plugin#47]
It would be nice if the plugin adjusted paths when a feature file was moved. Ideally, this should apply to `read` and `readonce` calls.

For example, when I moved a feature file into a subdirectory, a path was broken:
- This `* call read('this:../user/register.reusable.feature')`
- Should have changed to this `* call read('this:../../user/register.reusable.feature')`

I usually just drag and drop files around in IntelliJ's Project Tree, but for this, I also tested Move (F6), and the result was the same.

**Comment by ptrthomas** [comment]
@arnaldop absolutely. will keep this open until resolved. once we get references to be implemented correctly, refactoring should "just work"

**Comment by ptrthomas** [comment]
fixed in plugin version 3.0.1
